__tests__/run_seed.test.js

```diff
@@ -405,4 +405,26 @@ describe("PATCH", () => {
       });
   });
 });
-
+describe("DELETE : status 204 and no content", () => {
+  test("Deletes given comment by comment_id", () => {
+    return request(app)
+      .delete("/api/comments/5")
+      .expect(204)
+      .then(() => {
+        return request(app)
+          .get("/api/comments/5")
+          .expect(404)
+          .then(({ body: { msg } }) => {
+            expect(msg).toBe("Not found!");
+          });
+      });
+  });
+  test("Returns a 404 error message when passed a comment id that does not exist", () => {
+    return request(app)
+      .delete("/api/comments/12345")
+      .expect(404)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Not found!");
+      });
+  });
```

Comment:
Really good error testing so far but you could also add a test for when the id is invalid.